src/components/menu/menu.less

```diff
@@ -4,6 +4,9 @@
 // Ref: https://github.com/ariakit/ariakit/blob/454aef1237ada31e8292ee4966adca3ed0f6e299/packages/ariakit/src/menu/menu-list.ts#L150
 .reactist_menulist[role='menu'],
 .reactist_menulist[role='menubar'] {
+    min-height: 44px; // minimum accessible target area
+    max-height: var(--popover-available-height);
```

Comment:
Perhaps it makes sense to add a comment here indicating that `--popover-available-height` is defined by Ariakit?

Reply:
Yep! Makes complete sense.